examples/history-api-fallback/file

```diff
@@ -0,0 +1 @@
+file
```

Comment:
I have made common variable `contentBase` in `function Server` scope, I hope this is ok.